### PR TITLE
Publish to a "fast" worker

### PR DESCRIPTION
**Pull request by cgclever**
**[Associated SALT PR](https://github.com/Clever/salt/pull/2191/files)**

**Overview:**

Add configuration to publish a duplicate `s3-to-redshift-fast-prod` worker

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -9,7 +9,9 @@ notify:
 publish:
   catapult:
     url: $$catapult_url
-    application: "s3-to-redshift"
+    applications: 
+    - "s3-to-redshift"
+    - "s3-to-redshift-fast-prod"
```
Comment (nathanleiby) [inline .drone.yml]: nit: I'd name it just `fast` .. everything should support running in both dev and prod

**Comment by nathanleiby** [comment]
lgtm other than "prod" in name of worker

**Comment by evaninja** [comment]
LGTM, pending the naming comment that I made in the Salt PR

**Comment by kvigen** [comment]
Could you add a little background for why we split this into two workers to the PR description? Or possibly to the code?

**Comment by cgclever** [comment]
@kvigen: Thanks for asking. Updated the PR description.

**Comment by kvigen** [comment]
👍

**Pull request merged**